splash/qtrender_lua.py

```diff
@@ -1094,9 +1169,12 @@ def get_wrapped(self):
         return self._wrapped
 
     def run_async_command(self, cmd):
-        """ Execute _AsyncBrowserCommand """
-        meth = getattr(self.tab, cmd.name)
-        return meth(**cmd.kwargs)
+        """ Execute _AsyncBrowserCommand or _AsyncCallbackCommand"""
+        if isinstance(cmd, AsyncBrowserCommand):
+            meth = getattr(self.tab, cmd.name)
+            return meth(**cmd.kwargs)
+        elif isinstance(cmd, AsyncFunctionCommand):
+            cmd.kwargs["callback"]()
```

Comment:
Good catch. Usually async functions doesn't return any value and even if they do it isn't handled anywhere. 
I will return the `callback` value to make it look like the previous handler.